lib/easy_exports/data_loader.rb

```diff
@@ -22,6 +22,8 @@ def association_attributes(association_name)
                              end&.class_name
                            end
 
+        raise ArgumentError "Invalid Association: #{association_name}" unless association
```

Comment:
Is the intended expression "unless association_name"? However, in the present context, the use of "association" is beyond its scope.
The "association_attributes" method was called within the "fetch_records" method after "validating_selected_attributes". This method validates associations and triggers argument errors if they lack definitions. It was utilized in earlier gem versions. Despite its current inactivity, the method holds potential value for future versions which is why it's left there.